Wastewater_Seq_analysis/run_freyja_vrn_noBoot.sh

```diff
@@ -0,0 +1,85 @@
+#! /bin/bash
+# if any errors, then exit
+set -euxo pipefail
+
+###########################
+# Author: Pooja Gupta
+
+USAGE="
+Purpose:
+1) Run Freyja tool with wastewater sequencing data. 
+2) Retrieve BAM files for each sample generated by Viralrecon
+3) The input data are the bam files after ivar primer trimming step.
+Usage:
+run_freyja_vrn_noBoot.sh <wastewater sequencing run_name>
+Last updated on: June 13,2022
+"
+###########################
+
+echo $USAGE
+
+#Get Freyja tool (need to replace with Freyja's docker container)
+source /home/pgupta/miniconda3/etc/profile.d/conda.sh
```

Comment:
This is hardcoded to your username, this won't work for the production account

Reply:
Agree, that's why I commented that it should be replaced with a docker container :) I am assuming it would work then. I just don't know yet how to do it.